README.md

```diff
@@ -245,6 +245,17 @@ Run the following command from the project root to create a new dashboard admin:
 bundle exec rails 'create_admin_user[<name>,<email>,<password>]'
 ```
 
+### View Sandbox data in your local
+1. Create a new file in the [ansible/roles/common/files/ssh_keys/sandbox/<your-name>](https://github.com/simpledotorg/deployment/tree/master/ansible/roles/common/files/ssh_keys/sandbox) in the [deployment](https://github.com/simpledotorg/deployment) repo
+2. Run `pbcopy < ~/.ssh/id_rsa.pub` to copy your SSH key
+3. Paste the SSH key in the new file you created
+4. Create a new branch and submit a PR
+5. Wait for the PR to be reviewed, merged, and deployed to Sandbox
+6. Run `ssh deploy@ec2-13-235-33-14.ap-south-1.compute.amazonaws.com`
+7. Run `bundle exec cap sandbox db:pull` to sync the data with your local machine
+8. Request to be added as a Dashboard Admin to Sandbox
```

Comment:
You would want this to happen before syncing the data, right?

Reply:
I put the steps to request sandbox access before running the commands to sync data. That's what you meant right?